src/backend/parser/analyze.c

```diff
@@ -2986,6 +2986,27 @@ transformDistributedBy(ParseState *pstate, CreateStmtContext *cxt,
 
 	*policyp = policy;
 
+	if (cxt && cxt->inhRelations)
+	{
+		ListCell   *entry;
+
+		foreach(entry, cxt->inhRelations)
+		{
+			RangeVar   *parent = (RangeVar *) lfirst(entry);
+			Oid			relId = RangeVarGetRelid(parent, false, false /*allowHcatalog*/);
```

Comment:
Do you also want to comment on `false` as `false /*failOK*/` for easier code reading?